codespell_lib/data/dictionary.txt

```diff
@@ -37843,6 +37858,7 @@ unkonwns->unknowns
 unkown->unknown
 unkowns->unknowns
 unkwown->unknown
+unlabled->unlabeled
```

Comment:
`unlabled->unlabeled, unlabelled,`
for consistency with
`labled->labeled, labelled,`

Reply:
Yup, added.